### ISSUE TITLE
`-version` command does not print the version

### ISSUE THREAD

**Issue by silverwind** [editorconfig-checker/editorconfig-checker#337]
It only prints a single `\n`:

```bash
$ go run github.com/editorconfig-checker/editorconfig-checker/v2/cmd/editorconfig-checker@latest -version | xxd
00000000: 0a                                       .
```

**Comment by mstruebing** [comment]
After having a quick look I really think the whole version thing should be overhauled.
It's not that complicated but a lot of things can easily break: https://github.com/editorconfig-checker/editorconfig-checker/blob/main/Makefile#L89-L93

Maybe a good thing to do in conjunction with https://github.com/editorconfig-checker/editorconfig-checker/issues/204

**Comment by klaernie** [comment]
Oh - just noticed this: this is one of the cases of #383 - one would need to pass the version with `-ldflags` to make `go run …` work

**Comment by klaernie** [comment]
this is fixed since v3.1.0
